guides/_subsections/store-jmx-credentials.md

```diff
@@ -1,42 +1,46 @@
 ## [Store JMX Credentials](#store-jmx-credentials)
-If you have Java Management Extensions (JMX) authentication enabled on your containerized JVMs, Cryostat will prompt you to enter your JMX credentials before it can access the JDK flight recordings on your target JVMs. When monitoring multiple target JVMs with Cryostat features such as <a href="{{ page.url }}#create-an-automated-rule">Automatic Rules</a>, you may want Cryostat to remember and reuse your JMX credentials for each target connection. JMX credentials are automatically stored when you navigate to the <i>Recordings</i> tab and select a target JVM with JMX authentication enabled.
+If you have Java Management Extensions (JMX) authentication enabled on your containerized JVMs, Cryostat will prompt you to enter your JMX credentials before it can access the JDK flight recordings on your target JVMs. You can <a href="#configure-jmx-credentials-storage">Configure JMX Credentials Storage</a> and choose whether these credentials are held in browser memory for the current session only, or if they are uploaded to Cryostat's JMX credentials keyring.
 
-Cryostat stores JMX credentials according to each target’s unique JMX service URL, also known as a connection URL. Even if the underlying JVM instance changes, the target alias changes, or the target application restarts, Cryostat will apply the stored JMX credentials to the connection URL that the credentials are associated with.
-
-Here’s how to manage stored JMX credentials with the Cryostat web UI.
+If you intend to use Cryostat <a href="#create-an-automated-rule">Automated Rules</a> then you should store target application JMX credentials in Cryostat's JMX credentials keyring, which is outlined below. Even if you do not use Automated Rules you may find it more convenient to store credentials in the keyring, so that the Cryostat web UI does not need to prompt you for credentials when you manually access JFR information about target applications.
```

Comment:
Even if you do not use Automated Rules`,` you may find it ....

I would also consider ending the sentence on `credentials in the keyring`. `This way, the Cryostat web UI does not need...`